website/content/en/status/report-2025-07-2025-09/bananapi-r64-drivers.adoc

```diff
@@ -0,0 +1,31 @@
+=== FreeBSD Driver Development for BananaPi-R64
+
+Contact: Martin Filla <freebsd@sysctl.cz>
+Wiki:	 https://wiki.freebsd.org/arm/Bananapi
+
+=== Introduction
+The Banana Pi R64 is a MediaTek MT7622-based development board (ARM Cortex-A53, dual-core ~1.35 GHz) featuring 4× Gigabit LAN, 1× Gigabit WAN, Wi-Fi (4×4n), Bluetooth 5.0, and multiple peripheral interfaces (UART, SPI, I²C, GPIO, SATA, mini-PCIe, eMMC, etc.).
+
+=== Current State of FreeBSD Support
+- Implemented so far:
+  * **UART driver**
+  * **Clock management (clocks)**
+  * **Pinctrl/gpio driver** – in active development gpio part
+
+Other essential components—Ethernet, USB, SATA, Wi-Fi, etc.—are not yet implemented.
+
+=== Technical Context and Significance
+Banana Pi R64 is a flexible platform suited for embedded and router use, with rich peripheral support. Establishing FreeBSD compatibility would open possibilities for deploying FreeBSD in modern networking and appliance roles, especially given its reliability and network stack.
+
+=== Development roadmap
+* implement missing drivers
+  - Ethernet (GMAC ports)
+  - USB (XHCI/OTG)
+  - SATA / AHCI
+  - Storage controllers (eMMC/SD/MMC)
+  - Wi-Fi (likely MediaTek MT7615)
+  - GPIO subsystems
+
+=== Conclusion
+Support for Banana Pi R64 in FreeBSD is in the early stages—UART and clocks drivers exist but ppl clock is under develoment, gpio is under development—while most critical subsystems remain unimplemented. Publishing working code and artifacts, plus active collaboration with the FreeBSD community, will be essential to bring this board toward usable status under FreeBSD.
+
```

Comment:
Please remove final blank line. On the other hand, I would add a blank line after the `===` subtitles to improve file readability, but I do not think this is necessary for asciidoc, so it is only a suggestion.

```suggestion
```